### PR TITLE
Sync readme and helpfiles

### PR DESCRIPTION
**Pull request by maurolepore**
This PR aims to make it easier to keep README in sync with the helpfiles. Except for dropping the suffix `_chart()` in one function, none of this is crucial and you are welcome to disregard.

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -34,21 +58,30 @@ ggplot objects.
...
+```r
 # install.packages("devtools")
 devtools::install_github("2DegreesInvesting/r2dii.ggplot")
 ```
 
-## Examples using example data
+[How to minimize installation errors?](https://gist.github.com/maurolepore/a0187be9d40aee95a43f20a85f4caed6#installation)
 
-This is a basic example usage of `plot_trajectory_chart()`:
+## Example
```
Comment (maurolepore) [inline README.Rmd:68]: I recommend keeping the header exactly as "# Example" so we can expect a consistent url pointing to examples in all r2dii packages, for example:

https://2degreesinvesting.github.io/r2dii.data/#example
https://2degreesinvesting.github.io/r2dii.match/#example
https://2degreesinvesting.github.io/r2dii.analysis/#example

File: README.Rmd [hunk 2]
```diff
@@ -34,21 +58,30 @@ ggplot objects.
...
 library(r2dii.ggplot)
+```
+
+* `get_example_data()` `r pull_title("get_example_data")`.
+* `process_input_data()` `r pull_title("process_input_data")`.
 
+```{r}
 example_data <- get_example_data()
 example_data <- process_input_data(example_data)
+```
```
Comment (maurolepore) [inline README.Rmd:80]: Here is the helper `pull_title()` in action.

File: README.Rmd [hunk 3]
```diff
@@ -79,12 +112,11 @@ plot <- plot_trajectory(data_trajectory,
 plot
 ```
 
-This is a basic example usage of `plot_techmix_chart()`:
```
Comment (maurolepore) [inline README.Rmd:82]: `plot_techmix_chart()`  no longer has the suffix `_chart()`. Is this consistent across all functions?

**Review by maurolepore — COMMENTED** [review]
Here are some comments to clarify what I did. Except for dropping the suffix `_chart()` in one function, none of this is crucial and you are welcome to disregard.

File: README.Rmd (hunk 3 above)
Comment (MonikaFu) [reply README.Rmd:82]: Yes

**Review by MonikaFu — APPROVED** [review]
LGTM. Nice improvement, thank you Mauro!

**Pull request merged**